backend/app/mcp_server/tools/subscription.py

```
@@ -0,0 +1,878 @@
+    logger.info(
+        f"[MCP:Subscription] create_subscription called by user {token_info.user_id}, "
+        f"preview_id={preview_id}"
+    )
+
+    # Load preview data
+    preview_data = _get_preview_data(preview_id)
+    if not preview_data:
+        return json.dumps(
+            {
+                "success": False,
+                "error": "预览已过期或无效，请重新调用 preview_subscription 工具进行预览。",
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)